Sources/PackageLoading/Diagnostics.swift

```diff
@@ -87,6 +87,39 @@ extension Diagnostic.Message {
             resource file in the resources bundle
             """)
     }
+
+    static func copyConflictWithLocalizationDirectory(path: RelativePath, targetName: String) -> Self {
+        .error("resource '\(path)' in target '\(targetName)' conflicts with other localization directories")
+    }
+
+    static func missingDevelopmentRegion() -> Self {
+        .error("missing manifest property 'developmentRegion'; it is required in the presence of localized resources")
+    }
+
+    static func localizationAmbiguity(path: RelativePath, targetName: String) -> Self {
+        .error("""
+            resource '\(path)' in target '\(targetName)' is in a localization directory and has an explicit \
+            localization declaration; choose one or the other to avoid any ambiguity
+            """)
+    }
+
+    static func localizedAndUnlocalizedVariants(resource: String, targetName: String) -> Self {
+        .warning("""
```

Comment:
We can just make this an error, right?

Reply:
Using the same reasoning as the discussion bellow, it won't break anything to have both resources available: Foundation will just never choose one. So it's just a reminder to users. What do you think @neonichu ?

Reply:
`Foundation` provides `url(forResource:[...]localization:)` which would bypass it and grab the localized variant, so a user could theoretically have a use case for this situation. But it would be unorthodox, and they could just as easily put it somewhere else for their custom code to fetch from. Since the warning is extremely helpful 99% of the time, I would say keep the warning anyway.